packages/ra-ui-materialui/src/list/List.spec.tsx

```diff
@@ -340,4 +343,182 @@ describe('<List />', () => {
             screen.getByText('Books');
         });
     });
+
+    describe('"Select all" button', () => {
+        const Child = () => {
+            const [_, selectionModifiers] = useRecordSelection({
+                resource: 'books',
+            });
+            return (
+                <button onClick={() => selectionModifiers.clearSelection()}>
+                    reset
+                </button>
+            );
+        };
```

Comment:
Why not include it in the stories by default?

Reply:
because I think it's a bit ugly. Moreover, this button is useful for this test but not for the story

Reply:
That's useful to reproduce/debug the tests manually